ideascube/templatetags/ideascube_tags.py

```diff
@@ -130,3 +130,36 @@ def paginate(request, **kwargs):
     for key, value in kwargs.items():
         get[key] = value
     return '?{}'.format(get.urlencode())
+
+
+class AddGetParameter(template.Node):
+    def __init__(self, values):
+        self.values = values
+
+    def render(self, context):
+        req = template.resolve_variable('request',context)
+        params = req.GET.copy()
+        for key, value in self.values.items():
+            if key.endswith('s'):
+                for v in value:
+                    v = template.Variable(v).resolve(context)
+                    params.appendlist(key, v)
+            else:
+                params[key] = template.Variable(value).resolve(context)
+        return '?%s' %  params.urlencode()
+
+
+@register.tag()
```

Comment:
About naming, I'd go instead for `add_query_string` or `add_qs`.
`get` is a bit generic imho to have a clear meaning ;)